pkg/pkgmgmt/feed/feed.go

```diff
@@ -70,13 +73,22 @@ type MixinFileset struct {
 	Files   []*MixinFile
 }
 
-func (f *MixinFileset) FindDownloadURL(os string, arch string) *url.URL {
+func (f *MixinFileset) FindDownloadURL(ctx context.Context, os string, arch string) *url.URL {
+	log := tracing.LoggerFromContext(ctx)
+
 	match := fmt.Sprintf("%s-%s-%s", f.Mixin, os, arch)
 	for _, file := range f.Files {
 		if strings.Contains(file.URL.Path, match) {
 			return file.URL
 		}
 	}
+
+	// Until we have full support for M1 chipsets, rely on rossetta functionality in macos and use the amd64 binary
+	if os == "darwin" && arch == "arm64" {
+		log.Debugf("%s @ %s did not publish a download for darwin/amd64, falling back to darwin/amd64", f.Mixin, f.Version)
```

Comment:
A typo in "did not publish a download for darwin/amd64,...". Should it be "darwin/arm64"?